frontend/src/components/Model.ts

```diff
@@ -1,9 +1,10 @@
 
 
-export interface Cocktail {
+export interface CocktailFavs {
```

Comment:
`CocktailFavs` with an s in the end kind of indicates that the interface represents multiple favourites. That is not the case. It represents one cocktail.
In general I don't know why this interface exists. Isn't `FavouriteDrink` the interface for a favourite.